src/library/dlgtrackinfo.cpp

```diff
@@ -395,13 +393,27 @@ void DlgTrackInfo::saveTrack() {
         updatedRows.insert(oldRow);
 
         QVariant vHotcue = hotcueItem->data(Qt::DisplayRole);
-        if (vHotcue.canConvert<int>()) {
-            int iTableHotcue = vHotcue.toInt();
-            // The GUI shows hotcues as 1-indexed, but they are actually
-            // 0-indexed, so subtract 1
-            pCue->setHotCue(iTableHotcue - 1);
+        bool ok;
+
```

Comment:
These new lies don't help

Reply:
Why do you think that? Do you have any idea to do it better?

Reply:
Sorry, it was too late ;) The diff just shows two empty lines that could be removed to improve readability.

Reply:
I'll delete those empty lines myself after merging the PR ;)